libs/core/src/main/java/org/opensearch/common/io/stream/InputStreamStreamInput.java

```diff
@@ -80,7 +80,7 @@ public byte readByte() throws IOException {
     @Override
     public void readBytes(byte[] b, int offset, int len) throws IOException {
         if (len < 0) throw new IndexOutOfBoundsException();
-        final int read = Streams.readFully(is, b, offset, len);
```

Comment:
Should we mark this method in Streams as deprecated in favor of StreamsUtil until its cleaned up?  Although I see Streams as opensearch.internal, so thinking we can clean this up in a small follow up pr?

Reply:
I'll do a quick deprecation and cleanup here. That's a good idea.